app/views/administrate/application/_icons.html.erb

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+<svg hidden xmlns="http://www.w3.org/2000/svg">
```

Comment:
Uh, I didn't know of the `hidden` attribute. Cool, compatibility seems wide enough according to https://caniuse.com/?search=hidden. Only problematic with IE before 11, so it should be fine, I think.

Just one question: any reason that you went for this instead of a CSS-based solution? (As in: putting it in a stylesheet).

Reply:
With just a brief look around in the administrate css, there wasn't an obvious place and appropriate naming for me.
So this was simple, quick and elegant enough :)